src/chexus/validators.py

```diff
@@ -244,8 +244,11 @@ def validate(self, node: Dataset | Group) -> Violation | None:
         import scipp as sc
 
         rotation_speed = node.children.get("rotation_speed")
-        if "NXlog" == rotation_speed.attrs.get("NX_class"):
-            unit = rotation_speed.children.get("value").attrs.get("units")
+        if (
+            "NXlog" == rotation_speed.attrs.get("NX_class")
+            and "value" in rotation_speed.children
+        ):
+            unit = rotation_speed.children["value"].attrs.get("units")
         else:
             unit = rotation_speed.attrs.get("units")
```

Comment:
This now goes to the `else`, but a group should not have units, so this should be giving `unit=None`... which will show a violation?